tutorial_code/time/time_v1.toit

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the EXAMPLES_LICENSE file.
```

Comment:
Didn't you change the name of the license file?

Reply:
Yes. In a parallel PR.
I changed this one and the other ones I missed.
done.